Resources/Prototypes/Entities/Objects/Power/lights.yml

```diff
@@ -1,4 +1,26 @@
 ﻿- type: entity
+  parent: BaseItem
+  name: baselightbulb
+  id: BaseLightbulb
+  abstract: true
+  components:
+  - type: LightBulb
+  - type: Damageable
+  - type: Destructible
+    thresholds:
+      10:
+        behaviors:
+        - !type:PlaySoundCollectionBehavior
+          soundCollection: GlassBreak
+        - !type:SpawnEntitiesBehavior
```

Comment:
This is good but it would be better if you made a broken light bulb prototype and included it here (if you break it, it spawns 1 glass shard and 1 broken light bulb, if you break the broken light bulb it spawns 1 shard)

Reply:
I agree; rest looks good to me.

Reply:
@Zumorica had the (imo) better idea of not having separate prototypes for the broken states of the bulbs, and instead fixing the light bulb component's broken state, making it work with Breakable, and adding some sort of damage overlay visualizer for the broken state.

I think that might a bit out of scope for this PR, but I'd be interested in following up on it in the future when I'm more familiar.

Reply:
sounds good, you can solve the review then